pkg/rules/new.txt

```diff
@@ -0,0 +1,10 @@
+goos: darwin
```

Comment:
Maybe let's not commit the results of this, as that can be different machine to machine.